opencti-platform/opencti-front/src/private/components/settings/users/UserEditionOverview.tsx

```diff
@@ -102,24 +109,29 @@ UserEditionOverviewComponentProps
   const external = user.external === true;
   const objectOrganization = convertOrganizations(user);
 
-  const initialValues = pick(
-    [
-      'name',
-      'user_email',
-      'firstname',
-      'lastname',
-      'language',
-      'api_token',
-      'objectOrganization',
-      'description',
-      'account_status',
-      'account_lock_after_date',
-    ],
-    {
-      ...user,
-      objectOrganization,
-    },
-  );
+  const initialValues = {
+    // extract flat attributes
+    ...pick(
+      [
+        'name',
+        'user_email',
+        'firstname',
+        'lastname',
+        'language',
+        'api_token',
+        'objectOrganization',
+        'description',
+        'account_status',
+        'account_lock_after_date',
+      ],
+      {
+        ...user,
+        objectOrganization,
+      },
+    ),
+    // extract object attribute
+    max_confidence: user.user_confidence_level.max_confidence,
+  };
```

Comment:
Should we not get rid of this R.pick call completely ?

```suggestion
  const initialValues = {
    name: user.name,
    user_email: user.user_email,
    firstname: user.firstname,
    lastname: user.lastname,
    language: user.language,
    api_token: user.api_token,
    description: user.description,
    account_status: user.account_status,
    account_lock_after_date: user.account_lock_after_date,
    max_confidence: user.user_confidence_level.max_confidence,
    objectOrganization
  };
```

Reply:
you're right !